R/principal_components.R

```diff
@@ -420,10 +420,6 @@ principal_components.data.frame <- function(x,
   } else if (n >= ncol(x)) {
     n <- ncol(x) - 1
   }
-  # sanity check - we need at least two factors
```

Comment:
This was making that whenever we ran `factor_analysis(mtcars, n=1)` it stubbornely returned 2 factors...